components/Notification/Notification.tsx

```diff
@@ -0,0 +1,18 @@
+interface NotificationProps {
+  children: JSX.Element;
+  hasIcon?: boolean;
+}
+export const Notification = ({
+  children,
+  hasIcon = false,
```

Comment:
```suggestion
  hasIcon,
```
Esto no es del todo necesario, ya que si no le pasa la prop 'hasIcon', esta va a ser undefined y por ende 'falsy'

Reply:
En lo posible, me parece mejor dejarlo más verboso y no ir pasando undefineds que luego pueden tener problemas con typescript. Igual al final voy a sacar esa prop, porque no parece ser necesaria acá 😂